document.md

```diff
@@ -0,0 +1,21 @@
+# XML Variant Ingestion
+
+The ARGO Data Platform accepts variant calling data in XML format (based on hg19 genome reference). The XML file will be converted to VCF file followed by lift over to GRCh38 reference genome. For details, please see the latest version of the [ARGO xml_variant_ingestion](https://github.com/icgc-argo-workflows/dna-seq-processing-wfs/releases) workflow.
```

Comment:
Should we specific the variants produced are from FMI's pipeline? @lindaxiang thoughts?